### ISSUE TITLE
[Community contributed Expectations] - run_diagnostics() not working as `get_test_validator_with_data` reference cannot be found

### ISSUE THREAD

**Issue by maikelpenz** [fivetran/great_expectations#2563]
**Describe the bug**
The `run_diagnostics()` command used to validate/test [community contributed expectations](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations/how_to_contribute_a_new_expectation_to_great_expectations.html?highlight=Contribute) fails after [this PR](https://github.com/great-expectations/great_expectations/commit/90ed2ba8dc003b1c11581a64e280455d4bd08dad#) removed the local function `get_test_validator_with_data`. I can see this function exists inside `test_utils.py` but it is not imported on `self_check_util.py`

**To Reproduce**
1. Pick an existing community developed expectation (inside contrib/experimental/great_expectations_experimental/expectations) and run it.
2. The call to `self_check_report = ExpectColumnValuesToBeNormallyDistributed().run_diagnostics()` fails with the following error
```
    {
      "error_message": "name 'get_test_validator_with_data' is not defined",
      "stack_trace": "Traceback (most recent call last):\n  File \"Documents/Personal/projects/great_expectations/great_expectations/expectations/expectation.py\", line 803, in run_diagnostics\n    execution_engines,\n  File \"Documents/Personal/projects/great_expectations/great_expectations/expectations/expectation.py\", line 907, in _get_test_results\n    expectation_execution_engines_dict=execution_engines,\n  File \"Documents/Personal/projects/great_expectations/great_expectations/expectations/self_check_util.py\", line 1626, in generate_expectation_tests\n    validator_with_data = get_test_validator_with_data(\nNameError: name 'get_test_validator_with_data' is not defined\n"
    }
```

**Expected behavior**
Output validating that the expectation works and all tests passed

**Environment (please complete the following information):**
 - Operating System: MacOS
 - Great Expectations Version: 0.13.14

**Comment by eugmandel** [comment]
@maikelpenz Thank you for reporting this - we are working on a fix that will go out in the next minor release within a few days.

**Comment by eugmandel** [comment]
@alexsherstinsky That's the bug we talked about.

**Comment by alexsherstinsky** [comment]
@eugmandel Is this already working in our latest version merged into `develop` as of last night?  If so, then we can release today (or Monday) to unblock the user (next week would be preferred).  Thanks.

**Comment by eugmandel** [comment]
@alexsherstinsky let's do a minor version release early next week.

**Comment by eugmandel** [comment]
This is the PR that (among other things) contains the fix: https://github.com/great-expectations/great_expectations/pull/2567
